security/pass-otp/Portfile

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8::et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        tadfisher pass-otp 1.1.1 v
+revision            1
```

Comment:
In general `revision` is 0 for a new version, and 0 is the default, so the revision line is usually omitted.

Reply:
Ok, I'll drop this line then